pyccel/codegen/printing/cwrappercode.py

```diff
@@ -433,7 +433,7 @@ def _print_Module(self, expr):
 
         # Print imports last to be sure that all additional_imports have been collected
         imports  = [Import(s) for s in self._additional_imports]
-        imports += [Import('Python.h')]
+        imports += [Import('Python')]
         imports  = '\n'.join(self._print(i) for i in imports)
 
         return ('#define PY_SSIZE_T_CLEAN\n'
```

Comment:
It will be good if we include the header file instead of rewriting the functions signatures.

Reply:
I do not understand your comment. We do include the header file

Reply:
I mean in the generated wrapper file. instead of writing module functions signature, we only need to include the header generated.
I will give a detailed example.

Reply:
python module `pro.py`
```python
@types('int,int')
def rec(b,y):
    from math import abs
    return abs(b) + y
```
generated C files.
`pro.c`
```c
#include <pro.h>
/*........................................*/
long rec(long b, long y)
{

    long Out_0001;

    Out_0001 = fabs((double)(b)) + y;
    return Out_0001;
}
/*........................................*/
```
`pro.h`
```c
#ifndef pro_H // pro_H need to be PRO_H
#define pro_H // same here

#include <stdlib.h>
// we need also to include <math.h>

long rec(long b, long y);

#endif // pro_H
```
`pro_wrapper.c`
```c
#define PY_SSIZE_T_CLEAN
#include <stdlib.h> // we don't need this
#include <Python.h>

// here instead of writing signature we only need to include pro.h
long rec(long b, long y);
// we need to use separator only when the item printed
/*........................................*/



/*........................................*/

/*........................................*/
PyObject *rec_wrapper(PyObject *self, PyObject *args, PyObject *kwargs)
{

    long b;
    long y;
// more code
```

Reply:
Ah sorry, I had not realised we were talking about the wrapper file. I decided to handle the wrapper file differently as at some point in the near future it will also be used for fortran. When that happens it makes more sense to handle the difference in the function signatures in `cwrappercode.py` than to generate a `.h` file for a fortran function using `ccode.py` which is supposed to be pure c